<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timetabling Simutator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Home {
	background: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="home_layout">
   <item>
    <layout class="QHBoxLayout" name="header_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="logo_image">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>125</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../assets/klc_logo.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 35px;
	font-weight: bold;
	color: red;
	max-height: 100px;
}</string>
       </property>
       <property name="text">
        <string>University Timetabling Simulator</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="selection_vertical_layout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="select_label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: red;
	font-weight: bold;
	font-size: 18px;
	qproperty-alignment: AlignLeft;
	margin-top: 24px;
	margin-bottom: 2px;
}</string>
       </property>
       <property name="text">
        <string>Select Dataset:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="selection_horizontal_layout">
       <item>
        <widget class="QLabel" name="selected_dataset_label">
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: red;
	font-weight: bold;
	font-size: 18px;
	qproperty-alignment: AlignLeft;
	padding-top: 5px;
	border: 1px solid red;
	border-radius: 5px;
}</string>
         </property>
         <property name="text">
          <string>small_dataset.json</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse_button">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	max-width: 100px;
	max-height: 35px;
	color: white;
	font-weight: bold;
	background-color: red;
	border-radius: 5px;
}

QPushButton:hover:!pressed {
	background-color: #EC3812;
}</string>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="action_buttons_layout">
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	max-height: 35px;
	color: red;
	font-weight: bold;
	background-color: 	#D3D3D3;
	border-radius: 5px;
}

QPushButton:hover:!pressed {
	background-color: #DCDCDC;
}
</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_button">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	max-height: 35px;
	color: white;
	font-weight: bold;
	background-color: red;
	border-radius: 5px;
}

QPushButton:hover:!pressed {
	background-color: #EC3812;
}</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
